src/Infrastructure/LeanCode.Serialization/JsonConverters/JsonLaxDateTimeOffsetConverter.cs

```diff
@@ -0,0 +1,29 @@
+using System.Buffers;
+using System.Globalization;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace LeanCode.Serialization;
+
+public class JsonLaxDateTimeOffsetConverter : JsonConverter<DateTimeOffset>
+{
+    private static readonly UTF8Encoding encoder = new(false);
+    private readonly string format = "yyyy'-'MM'-'dd'T'HH':'mm':'ss.fffZ";
+    public override DateTimeOffset Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        var value = reader.ValueSpan;
+        Span<char> chars = stackalloc char[value.Length];
+        encoder.GetChars(value, chars);
+        if (DateTime.TryParse(chars, out DateTime date))
```

Comment:
Aren't you losing the offset data by parsing with `DateTime` instead of `DateTimeOffset` directly?